### ISSUE TITLE
Discussions

### ISSUE THREAD

**Issue by ocarm** [dell/PyU4V#117]
Hi, 
Sorry i have not found how to open a discussion. Is there any plan to implement remoteSymmSGInfoParam when adding or deleting a volume in a storage or  -keepR1 param to delete a volume in a SRDF Metro pairing ?

Exemple :
```
{
  "editStorageGroupActionParam": {
    "expandStorageGroupParam": {
      "addSpecificVolumeParam": {
        "volumeId": [
          "000F8"
        ],
        "remoteSymmSGInfoParam": {
          "force": true
        }
```

**Comment by rawstorage** [comment]
Olivier, I can have a look, in the meantime if you use the method https://pyu4v.readthedocs.io/en/latest/quick_start.html any functionality that is missing in the SDK can be overcome by creating your own method.  

https://pyu4v.readthedocs.io/en/latest/quick_start.html#perform-a-custom-rest-call-in-pyu4v

**Comment by ocarm** [comment]
Hi Paul,

Thanks, i will look at these functions.

**Comment by rawstorage** [comment]
Oliver, I'm finally getting around to look at these changes, which functions were you specifically asking for the keepR1 Parameter?

**Comment by ocarm** [comment]
in replication.py : **delete_storage_group_srdf** 
when you want to remove a volume or break a pairing in a SRDF/M rdf group.

The keepR1 pquery parameter that i'm talking about  :
/91/replication/symmetrix/{symmetrixId}/storagegroup/{storageGroupId}/rdf_group/{rdfgNumId}

DELETE

delete SRDF pairing for a particular RDFG
Parameters
name | description | type | default
-- | -- | -- | --
symmetrixId | the local array for the RDF setup | path |  
storageGroupId | the local SG for the RDF setup | path |  
rdfgNumId | the RDF group number that will be removed | path |  
half_delete | is the command a Half Delete (as opposed to full) | query |  
hop2 | is this to run on the Hop2 leg | query |  
force | is the force option to be used | query |  
symforce | is the symforce option to be used, use this with care | query |  
star | are there volumes in a Star configuration | query |  
bypass | (no documentation provided) | query |  
**keepR1** | (no documentation provided) | query |
keepR2 | (no documentation provided) | query |

**Comment by rawstorage** [comment]
so adding the options to delete_storage_group_srdf should be enough to acomplish this, if I add a filters option where you can pass these as a dictionary, this should do the trick for you?

**Comment by rawstorage** [comment]
suggested implementation would allow for all,
 :param filters: optional boolean filters, half_delete,hop2, force,
                        symforce,star,bypass,keepR1, keepR2, 
                        e.g. filters={'keepR1': 'true'}  -- dict

**Comment by ocarm** [comment]
It seems fine to me. Thanks !

**Comment by rawstorage** [comment]
Olivier, 
I have these changes added to https://github.com/dell/PyU4V/tree/92/consistency_disable branch as discussed.  Can you have a look and see if you like the change.

**Comment by ocarm** [comment]
Paul, 
I have currently  no unisphere 9.2 environnent at my disposal so I've only look at the code. It seems to fullfill my expectation.

**Comment by rawstorage** [comment]
Fix delivered in 9.2.1.4 just released
